main.go

```diff
@@ -120,11 +121,15 @@ type AppState struct {
 	Settings
 	ArborState
 	*sprigTheme.Theme
-	DataDir string
+	DataDir      string
+	TimeLaunched uint64
 }
 
 func NewAppState(dataDir string) (*AppState, error) {
 	dataDir = filepath.Join(dataDir, "sprig")
+	// NOTE: time has to be converted from nanoseconds to milliseconds for Arbor's specifications
+	TimeLocation, _ := time.LoadLocation("UTC")
+	TimeLaunched := uint64(time.Now().In(TimeLocation).UnixNano() / 1000000)
```

Comment:
I had to force the time library to use UTC and then manually convert nanoseconds to milliseconds.

Reply:
I appreciate the effort that you put in here, but you don't need to mess with the time zone. Unix timestamps are always relative to UTC.

[Here](https://play.golang.org/p/8xQyigxhy4N) is a program demonstrating that the result of `UnixNano()` doesn't vary by time zone. The current time in a playground is fixed though, so copy that and run it on your local machine if you want to be extra sure.

I believe you can simplify this to:
```suggestion
	TimeLaunched := uint64(time.Now().UnixNano() / 1000000)
```

Reply:
Oh. I only tested the difference with time.Now() and thought it was needed. Thank you for clarifying the time API.